tests/benchq/algorithms/test_lde_solver.py

```diff
@@ -125,3 +126,24 @@ def test_inverse_blockencoding(beta, epsilon, time):
     assert isinstance(sel_inv, Circuit)
     assert sel_inv.n_qubits == total_qubits
     assert expected_num_of_custom_gates == actual_num_of_custom_gates
+
+
+matrix_norm, dummy_circuit = construct_dummy_cir()
+matrix_exp_test = matrix_exponentiation(dummy_circuit, matrix_norm, 0.1, 1.3, 1e-1)
```

Comment:
Just a minor comment: it would be good to make it clear that the values of beta and time used here should match those used on lines 145. One way to do this would be to move the definitions of `beta` and `time` from lines 140-141 outside of the unit test and use these variables on line 132.